.travis.yml

```diff
@@ -7,6 +7,11 @@ d:
   #- gdc
   - ldc
 
+env:
+  - VIBE_VERSION='--version=0.7.31'
+  - VIBE_VERSION='--version=0.8.0'
+  - VIBE_VERSION=''
```

Comment:
included this so we test once with the latest version, in case the above ones get out of date